packages/storybook-addon-baseline-grid/.storybook/main.ts

```diff
@@ -0,0 +1,18 @@
+import type { StorybookConfig } from "@storybook/react-vite";
+const config: StorybookConfig = {
+  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  addons: [
```

Comment:
```suggestion
  core: {
    disableTelemetry: true,
  },
  addons: [
```
Bit of a minor nitpick, but can we disable the storybook usage telemetry here? We are [already disabling it](https://github.com/canonical/ds25/blob/abf6ee9b1b22a7d1ffa4c189c9ae503a6cc9bd6e/packages/ds-react-core/.storybook/main.ts#L28) in the react core package.

Reply:
Done